### PR TITLE
Jaeger: Make search form fields more consistent with Tempo

### PR DESCRIPTION
**Pull request by CrypticSignal**
**What this PR does / why we need it**:
This PR makes Jaeger's search form more consistent with Tempo's search form. More specifically:
- The `Service` field has been renamed to `Service Name`, and the placeholder has been changed to `Select a service`.
- The `Tags` field now includes a tooltip like it does in Tempo's search form.

### CODE REVIEW

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 1]
```diff
@@ -144,7 +145,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
         </InlineField>
       </InlineFieldRow>
       <InlineFieldRow>
-        <InlineField label="Tags" labelWidth={14} grow>
+        <InlineField label="Tags" labelWidth={14} grow tooltip="Values should be in the logfmt format.">
```
Comment (joey-grafana) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:148]:
```suggestion
        <InlineField label="Tags" labelWidth={14} grow tooltip="Values should be in logfmt.">
```

**Review by joey-grafana — CHANGES_REQUESTED** [review]
Could you also add a placeholder for the operation select? And update the label to be Operation Name

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx (hunk 1 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/SearchForm.tsx:149]: Done! https://github.com/grafana/grafana/pull/54026/commits/a81bf187cd54179b2bee788261c71d9f591b633e

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 2]
```diff
@@ -130,6 +131,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
             }
             isLoading={isLoading.operations}
             value={operationOptions?.find((v) => v.value === query.operation) || null}
+            placeholder="Select operation name"
```
Comment (joey-grafana) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:134]: Can you change this to select an operation

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 3]
```diff
@@ -144,7 +146,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
         </InlineField>
       </InlineFieldRow>
       <InlineFieldRow>
-        <InlineField label="Tags" labelWidth={14} grow>
+        <InlineField label="Tags" labelWidth={14} grow tooltip="Values should be in the logfmt format.">
```
Comment (joey-grafana) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:149]: You may have missed my suggestion here: tooltip should be `Values should be in logfmt.`

**Review by joey-grafana — CHANGES_REQUESTED** [review]

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx (hunk 3 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/SearchForm.tsx:149]: The reason I wrote it the way I did is to be consistent with the tooltip for the same field in Tempo. So I'd like to confirm whether you'd like me to induce this inconsistency?

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx (hunk 2 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/SearchForm.tsx]: Done. https://github.com/grafana/grafana/pull/54026/commits/43fa1ac8e2ad12f054c5d07021b41222819c65ae

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx (hunk 3 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/SearchForm.tsx]: Done. https://github.com/grafana/grafana/pull/54026/commits/e55a5a64924a182d5a1ff9c4d9419d05b5f36fc5

**Review by joey-grafana — COMMENTED** [review]
LGTM 👍

**Review by joey-grafana — APPROVED** [review]

**Pull request merged**